src/sidebar/templates/group-list.html

```diff
@@ -1,22 +1,26 @@
 <div class="pull-right"
      dropdown
      keyboard-nav>
+  <!-- Show a drop down menu if showGroupsMenu is true. -->
   <div class="dropdown-toggle"
         dropdown-toggle
-        data-toggle="dropdown"
-        role="button"
-        title="Change the selected group">
+        data-toggle="dropdown">
     <img class="group-list-label__icon group-list-label__icon--organization"
          ng-src="{{ vm.focusedIcon() }}"
          alt="{{ vm.orgName(vm.groups.focused().id)}}"
          ng-if="vm.focusedIcon()">
          <i class="group-list-label__icon h-icon-{{ vm.focusedIconClass() }}"
              ng-if="!vm.focusedIcon()"></i><!-- nospace
-    !--><span class="group-list-label__label">{{vm.groups.focused().name}}</span><!-- nospace
-      !--><i class="h-icon-arrow-drop-down"></i>
+    !-->
+    <h2 class="group-list-label__label has-toggle" ng-if="vm.showGroupsMenu()"><a class="group-list-label__toggle">{{vm.groups.focused().name}}</a><!-- nospace
+      !--><i class="h-icon-arrow-drop-down" ng-if="vm.showGroupsMenu()"></i></h2>
```

Comment:
If `ng-if` on the `<i>` looks unnecessary to me since the same conditional exists on the parent element. If the parent element is hidden, the child will not be visible anyway.

Reply:
Yeah we were just messing around. I just committed whatever i had for @dwolfe to continue playing with it.